### PR TITLE
Resolved issue #117

### PR DESCRIPTION
**Pull request by AliAzaz**
Confirm delete prompt shown in CameraXBasic even when no photo available. #117 Done

### CODE REVIEW

**Comment by AliAzaz** [comment]
Please check the code. I implemented the better way to use kotlin for this issue.

**Comment by AliAzaz** [comment]
Kindly do check this @owahltinez

File: CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt [hunk 1]
```diff
@@ -149,10 +151,13 @@ class GalleryFragment internal constructor() : Fragment() {
...
-                        }}
 
-                    .setNegativeButton(android.R.string.no, null)
-                    .create().showImmersive()
+                        }
+
+                        .setNegativeButton(android.R.string.no, null)
+                        .create().showImmersive()
+
+            } ?: Toast.makeText(activity, "No media file available", Toast.LENGTH_SHORT).show()
```
Comment (piotrek1543) [inline CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt]: I am not owner of this repository, but could you extract it to `strings.xml`?

File: CameraXBasic/gradle.properties [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-android.useAndroidX=true
-android.enableUnitTestBinaryResources=true
+# Default value: -Xmx1024m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+#Tue Dec 24 15:36:07 PKT 2019
```
Comment (piotrek1543) [inline CameraXBasic/gradle.properties]: I wonder how it is related to fixed issue :-) I think it should be in another merge request with updated gradle build tools version ;-)

Comment (AliAzaz) [reply CameraXBasic/gradle.properties]: Sir what I implemented is better way to solve in Kotlin and if your intention is to only resolve the issue then close this pull request.

File: CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt (hunk 1 above)
Comment (AliAzaz) [reply CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt]: Sir I only resolved the issue that is mentioned in issue title. Secondly, yes we can do extract.

**Comment by AliAzaz** [comment]
@owahltinez kindly do check it

File: CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt (hunk 1 above)
Comment (piotrek1543) [reply CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/GalleryFragment.kt]: :+1:

File: CameraXBasic/gradle.properties (hunk 2 above)
Comment (piotrek1543) [reply CameraXBasic/gradle.properties]: :ok_hand:

**Comment by owahltinez** [comment]
Hi @AliAzaz thanks for this pull request! Can you help me understand how this is different from what is being proposed in #118 ?

**Comment by AliAzaz** [comment]
I defined why I created this PR in Issue#134 comments

**Comment by owahltinez** [comment]
@AliAzaz I'm closing this one in favour of #118 which solves the same problem, but that one was submitted first. If #118 is not merged soon, feel free to reopen this one and address the requested changes.

**Comment by AliAzaz** [comment]
Sir the way I do code to resolve this issue is better way to work in Kotlin

**Pull request closed without merging**